content/md/articles/language/core_overview.md

```diff
@@ -70,13 +80,28 @@ pages.
 ```
 
 `def` takes a symbol, an optional docstring, and an optional init value
-(although it is very rare that you would omit the init value).
+(although it is very rare that you would omit the init value), and creates (if it doesn't already exist) a var in the current namespace with the name of the symbol.
```

Comment:
That wording makes it sound like the following would leave `x` with the value `1` since "it already exists" when the second `def` is evaluated:
```clojure
(def x 1)
(def x 2)
```
Not quite sure what to suggest but maybe you can think of slightly different wording that makes it clear the `Var` creation is separate from the binding to a value?

Reply:
I tweaked this wording to more comprehensively describe all the scenarios